custom_random_forest.py

```diff
@@ -0,0 +1,56 @@
+from sklearn.base import BaseEstimator
+import numpy as np
+from multiprocessing import Pool
+from sklearn.tree import DecisionTreeClassifier
+
+class RandomForestClassifierCustom(BaseEstimator):
+    def __init__(
+        self, n_estimators=10, max_depth=None, max_features=None, random_state=None
+    ):
+        self.n_estimators = n_estimators
+        self.max_depth = max_depth
+        self.max_features = max_features
+        self.random_state = random_state
+        self.trees = []
+        self.feat_ids_by_tree = []
+
+    def fit_tree(self, args):
+        i, X, y = args
+        np.random.seed(self.random_state)
```

Comment:
```suggestion
        i, X, y = args
        np.random.seed(self.random_state + i)
```
Чтобы деревья отличались, но все было воспроизводимо